sim/common/wotlk/shadowmourne.go

```diff
@@ -49,21 +49,22 @@ func init() {
 					return
 				}
 
+				if stackingAura.GetStacks() == 10 {
+					stackingAura.Deactivate(sim)
+					tempStrProc.Activate(sim)
+					choasBaneSpell.Cast(sim, player.CurrentTarget)
+				}
```

Comment:
return? Also wont this currently start proccing the new stacks before the big str proc is faded? Should that be possible?

Reply:
i went back and forth on it but I am not sure how it will work. We need to test but obv can't right now. The dps difference is like 10 dps.